python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_schedule_invocation.py

```diff
@@ -15,12 +13,6 @@
 from dagster.core.test_utils import instance_for_test
 
 
-def test_schedule_context_backcompat():
-    # If an instance of ScheduleEvaluationContext is a ScheduleExecutionContext, then annotating as
-    # ScheduleExecutionContext and passing in a ScheduleEvaluationContext should pass mypy
-    assert isinstance(ScheduleEvaluationContext(None, None), ScheduleExecutionContext)
```

Comment:
it would be surprising to accidentally delete since theres a comment next to it - but you could preserve a test here that just annotates some functions as am additional guard

Reply:
I actually realize now that I made a mistake in making `ScheduleExecutionContext` a type alias, because it's a literal alias of `ScheduleEvaluationContext` that's exported top-level. I guess we'll remove with 1.0 but for now I'll restore it (and the test), though the comment I think is wrong, we're not testing if mypy passes here but rather python `isinstance`